src/AzureAppConfigurationImpl.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { AppConfigurationClient, ConfigurationSetting, ConfigurationSettingId, GetConfigurationSettingOptions, GetConfigurationSettingResponse, ListConfigurationSettingsOptions, isFeatureFlag } from "@azure/app-configuration";
+import { AppConfigurationClient, ConfigurationSetting, ConfigurationSettingId, FeatureFlagValue, GetConfigurationSettingOptions, GetConfigurationSettingResponse, ListConfigurationSettingsOptions, featureFlagPrefix, isFeatureFlag, parseFeatureFlag } from "@azure/app-configuration";

```

Comment:
This is coming from JS SDK but I'm surprised to see some of these in pascal case and others in camel case. Do you know why that's the case?

Reply:
In JS the naming convention is usually as below:
- CONSTANT_VARIABLE_NAME
- TypeName
- variableName
- methodName 

In JS SDK they regard constants as variables and use constantVariableName. Makes sense to some extent.
In this repo, some constants are named as ConstantVariableName when copied from dotnet for reference, which I think should eventually be changed.

Reply:
unify constant naming in #68

Reply:
constants in this PR renamed in 708f759